lib/vector-core/src/event/proto.rs

```diff
@@ -1,11 +1,11 @@
 use std::collections::BTreeMap;
 use std::sync::Arc;
 
-use chrono::TimeZone;
-use ordered_float::NotNan;
-
 use super::{MetricTags, WithMetadata};
 use crate::{event, metrics::AgentDDSketch};
+use chrono::TimeZone;
+use ordered_float::NotNan;
+use uuid::Uuid;
```

Comment:
I'm going to assume this is an editor setting, but these (and other files) ended up being reordered differently than we customarily have them.

Reply:
Ah I think auto-importing `Uuid` then running `cargo fmt` does this re-ordering, will keep an eye out for this in the future